src/o365/spo/commands/contenttype/contenttype-field-remove.ts

```diff
@@ -0,0 +1,307 @@
+        }
+        else {
+          requestBody =  `<Request AddExpandoFieldTypeSuffix="true" SchemaVersion="15.0.0.0" LibraryVersion="16.0.0.0" ApplicationName=".NET Library" xmlns="http://schemas.microsoft.com/sharepoint/clientquery/2009"><Actions><ObjectPath Id="77" ObjectPathId="76" /><ObjectPath Id="79" ObjectPathId="78" /><Method Name="DeleteObject" Id="80" ObjectPathId="78" /><Method Name="Update" Id="81" ObjectPathId="24"><Parameters><Parameter Type="Boolean">${updateChildContentTypes}</Parameter></Parameters></Method></Actions><ObjectPaths><Property Id="76" ParentId="24" Name="FieldLinks" /><Method Id="78" ParentId="76" Name="GetById"><Parameters><Parameter Type="Guid">{${args.options.fieldLinkId}}</Parameter></Parameters></Method><Identity Id="24" Name="6b3ec69e-00a7-0000-55a3-61f8d779d2b3|740c6a0b-85e2-48a0-a494-e0f1759d4aa7:site:${this.siteId}:web:${this.webId}:contenttype:${args.options.contentTypeId}" /></ObjectPaths></Request>`
+        }
+        const requestOptions: any = {
+          url: `${args.options.webUrl}/_vti_bin/client.svc/ProcessQuery`,
+          headers: Utils.getRequestHeaders({
+            authorization: `Bearer ${this.siteAccessToken}`,
+            'X-RequestDigest': this.requestDigest
+          }),
+          body: requestBody
+        };
+
+        if (this.debug) {
+          cmd.log('Executing web request.');
+          cmd.log(requestOptions);
+          cmd.log('');
+        }
+
+        return request.post(requestOptions);
+      })
+      .then((res: any): void => {
+        if (this.debug) {
+          cmd.log('Response:');
+          cmd.log(res);
+          cmd.log('');
+        }
+
+        const json: ClientSvcResponse = JSON.parse(res);
+        const response: ClientSvcResponseContents = json[0];
+        if (response.ErrorInfo) {
+          cb(new CommandError(response.ErrorInfo.ErrorMessage));
+          return;
+        }
+        if (this.verbose) {
+          cmd.log(vorpal.chalk.green('DONE'));
+        }
+        cb();
+      }, (error: any): void => {
+        this.handleRejectedODataJsonPromise(error, cmd, cb);
+      });
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.contentTypeId = (!(!args.options.contentTypeId)).toString();
```

Comment:
`contentTypeId` is a required property so this will always resolve to true. Instead we should track the usage of optional properties such as `listTitle` or `updateChildContentTypes`.